src/GovUk.Education.ExploreEducationStatistics.Content.Services/ViewModels/ThemeViewModel.cs

```diff
@@ -1,5 +1,12 @@
 ﻿#nullable enable
 
+using System;
+
 namespace GovUk.Education.ExploreEducationStatistics.Content.Services.ViewModels;
 
-public record ThemeViewModel(string Title);
+public record ThemeViewModel
```

Comment:
Was there a reason not to go for the inline constructor?

Reply:
No there wasn't, I've changed it to use the inline primary constructor now at the same time as  adding `Summary` which @amyb-hiveit asked for. That's on a new commit. Thanks 🙂